base/set.jl

```diff
@@ -145,6 +145,23 @@ setdiff!(s::Set, xs) = (for x=xs; delete!(s,x); end; s)
 <( l::Set, r::Set) = (length(l) < length(r)) && (l <= r)
 <=(l::Set, r::Set) = issubset(l, r)
 
+"""
+    issubset(a, b)
+    ⊆(a,b) -> Bool
+    ⊈(a,b) -> Bool
+    ⊊(a,b) -> Bool
```

Comment:
this has been this way since #7918, but little odd that the reversed-sense versions of the unicode operator are documented as if they're synonyms... (also the case for `in`)